<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Nehas-MacBook-Air.local" failures="1" tests="8" name="TestLevel" time="95.704" errors="0" timestamp="24 May 2021 09:43:41 GMT">
  <testcase classname="TestCase1" name="Test1" time="9.025"/>
  <testcase classname="TestCase2" name="Test2" time="10.219"/>
  <testcase classname="TestCase2" name="Test2" time="21.294"/>
  <testcase classname="TestCase2" name="Test2" time="14.845"/>
  <testcase classname="TestCase2" name="Test2" time="11.888"/>
  <testcase classname="TestCase2" name="Test2" time="10.798"/>
  <testcase classname="TestCase3" name="Test3" time="8.193"/>
  <testcase classname="TestCase4" name="Test3" time="9.383">
    <failure type="java.lang.AssertionError" message="expected [$21.98] but found [$10.99]">
      <![CDATA[java.lang.AssertionError: expected [$21.98] but found [$10.99]
at TestCase4.Test3(TestCase4.java:63)
at jdk.proxy1/jdk.proxy1.$Proxy2.stop(Unknown Source)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
at java.base/java.lang.Thread.run(Thread.java:831)
... Removed 52 stack frames]]>
    </failure>
  </testcase> <!-- Test3 -->
</testsuite> <!-- TestLevel -->
